apps/web/src/components/Alert.tsx

```diff
@@ -3,12 +3,25 @@ import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
 import { ReactNode } from 'react';
 
 type AlertProps = {
+  color?: 'blue' | 'yellow' | 'red';
   children: ReactNode;
 };
 
-export const Alert = ({ children }: AlertProps) => {
+export const Alert = ({ color, children }: AlertProps) => {
+  let containerClasses = '';
+  switch (color) {
+    case 'red':
+      containerClasses = 'bg-bcRedError py-4  text-left flex items-center rounded';
+      break;
+    case 'yellow':
+      containerClasses = 'bg-bcYellowCream py-4  text-left flex items-center rounded';
+      break;
+    default:
+      containerClasses = 'bg-bcLightBlueBackground py-4  text-left flex items-center rounded';
+  }
+
   return (
```

Comment:
I may refactor to this, but up to you :)

```tsx
import classNames from 'classnames';

let containerClasses = '';
const baseClasses = 'py-4 text-left flex items-center rounded';

switch (color) {
  case 'red':
    containerClasses = classNames(baseClasses, 'bg-bcRedError');
    break;
  case 'yellow':
    containerClasses = classNames(baseClasses, 'bg-bcYellowCream');
    break;
  default:
    containerClasses = classNames(baseClasses, 'bg-bcLightBlueBackground');
}
```

Reply:
That is much cleaner, thank you!